### ISSUE TITLE
Setup-TraefikContainerForNavContainers.ps1: add param to define environment variables

### ISSUE THREAD

**Issue by ti-jalopez** [microsoft/navcontainerhelper#1574]
**Using acme provider OVH requires environment variables: add param for environment variables**
Setup-TraefikContainerForNavContainers don't allow use the param -e in "docker run" command, and some acme providers, such as OVH, requires defining some environment variables in traefik container.
Please, add a new param to pass a list of variables and values, to use in docker run command.

**Scripts used to create container and cause the issue**
```
$publicDnsName = 'my.public.dom'
$publicLocalIP = '192.168.100.100'
$emailDnsACME = 'emal@mydomain.es'

Setup-TraefikContainerForBCContainers `
  -PublicDnsName $publicDnsName -IP $publicLocalIP -ContactEMailForLetsEncrypt $emailDnsACME `
  -overrideDefaultBinding -forceHttpWithTraefik
```

**Manual fix**
There is no error running the script, but I need to do this steps for allow to use OVH provider:
1) Delete traefik container created by Setup-TraefikContainerForBCContainers
2) Modify traefik.toml to add some parametres for OVH provider:
```
[acme.dnsChallenge]
provider = "ovh"
delayBeforeCheck = 0
resolvers = ["8.8.8.8:53"]
```
3) Create traefik container adding environment variables (-e param):
```
$traefikForBcBasePath = "C:\ProgramData\BcContainerHelper\traefikforbc"
$traefikDockerImage = "stefanscherer/traefik-windows:v1.7.12"
$traefikContainerName = "traefik"

docker run `
  -e OVH_ENDPOINT=my-ovh-endpoing `
  -e OVH_APPLICATION_KEY=my-ovh-key `
  -e OVH_APPLICATION_SECRET=my-ovh-secret `
  -e OVH_CONSUMER_KEY=my-ovh-cons-key `
  -p 443:443 -p 80:80 --restart always --name $traefikContainerName --hostname $traefikContainerName -d `
  -v ((Join-Path $traefikForBcBasePath "config") + ":c:/etc/traefik") `
  -v \\.\pipe\docker_engine:\\.\pipe\docker_engine $traefikDockerImage --docker.endpoint=npipe:////./pipe/docker_engine 
```

**Screenshots**

**Additional context**
See previous issue, to allow exclude some ports for mapping in https://github.com/microsoft/navcontainerhelper/issues/1573

**Comment by freddydk** [comment]
In the next bccontainerhelper preview there is a -additionalParameters parameter on the setup- function.

**Comment by ti-jalopez** [comment]
Great @freddydk,
I'll test it when the new version is available.
Best regards,

**Comment by freddydk** [comment]
Shipped in 1.0.18
